app/views/devise/registrations/new.html.haml

```diff
@@ -0,0 +1,36 @@
+.container
+  .row.justify-content-center
+    .col-md-4
+      %h2.header= t(".sign_up")
+      = form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f|
+        = devise_error_messages!
+        .form-group
+          = f.label :email
+          %br
+          = f.email_field :email, autofocus: true, autocomplete: "email", class: "form-control"
+        .form-group
+          = f.label :name
+          %br
+          = f.text_field :name, class: "form-control"
+        .form-group
+          = f.label :surname
+          %br
+          = f.text_field :surname, class: "form-control"
+        .form-group
+          = f.label :phone_number
+          %br
+          = f.text_field :phone_number, class: "form-control"
+        .form-group
+          = f.label :password
+          - if @minimum_password_length
+            %em
+              = t("#{@minimum_password_length} characters minimum")
+          %br
+          = f.password_field :password, autocomplete: "off", class: "form-control"
+        .form-group
+          = f.label :password_confirmation
+          %br
+          = f.password_field :password_confirmation, autocomplete: "off", class: "form-control"
+        .actions
+          = f.submit t(".sign_up"), class: "btn btn-outline-info"
+      = render "devise/shared/links"
```

Comment:
Do you have this partial already merged?

Reply:
No, this one Nastia did. So I deleted file (partial) and added only this line

Reply:
If you merge before Nastia, it will crash
You can't add this

Reply:
rebase from Nastia?

Reply:
without partial file it use "gem devise", after merge it will use Nastia`s partial, is it?
In my view it shouldn't crash.